runtime/near-vm-logic/Cargo.toml

```diff
@@ -17,10 +17,13 @@ base64 = "0.13"
 borsh = "0.8.1"
 bs58 = "0.4"
 byteorder = "1.2"
+num-integer = "0.1"
+ripemd160 = "0.9.0"
 serde = { version = "1", features = ["derive"] }
 sha2 = ">=0.8,<0.10"
 sha3 = ">=0.8,<0.10"
 
+near-crypto = { path = "../../core/crypto" }
```

Comment:
FYI, this makes near-vm-logic not compilable to Wasm, which is needed for AS SDK and near-sdk-sim. It would require some ugly refactoring to avoid doing it. Given that we are replacing simulator and mock-vm in our contract testing infrastructure with the sandbox it is not problematic, but Developer Platform team needs to be aware of it. CC @mikedotexe @austinabell @willemneal @DiscRiskandBisque @ChaoticTempest 

This PR will be a forcing function to move all testing infrastructure to the sandbox.

Reply:
but if this functionality is put under a feature flag as mentioned elsewhere in this PR, this wouldn't be required for sim/unit testing in the SDK, would it? (just going off intuition based on scanning the PR, can dig deeper to understand later)

Reply:
It will be stabilized soon

Reply:
This isn't an issue for `sdk-sim` as the host side is still compiled to native.  However, a feature flag for non-wasm features would suffice.

@abacabadabacaba Is there any way to make the crypto code compile to Wasm just for testing purposes?

Reply:
@willemneal Why doesn't it compile now? Ideally it should be entirely portable.

Reply:
Hm. Shoot. Would I need to move all this logic directly then instead of from `near-crypto`?

Reply:
I haven't tried, but given this comment, I thought that was Max's implication.  In the past some crypto code relies on inline assembly.

Reply:
He had said that it should be ok in the end on Slack, unless someone objects.

Reply:
@austinabell When features stabilize they become enabled by default and then they also need to be compiled to Wasm. However, as I explained above we should stop compiling nearcore code to Wasm.